### DISCUSSION TITLE
Hyperlink not possible via the API

### DISCUSSION THREAD

**Discussion by Denon9205** [bluesky-social/atproto#1587] [How to do it?] [answered]
I can't get the link to display as a hyperlink via API. Posting a message works without any problem. I used the following documenation:
[atproto.com/blog/create-...](https://atproto.com/blog/create-post)

https://bsky.app/profile/dinowithlc.bsky.social/post/3k76e6mqnba2k

**Comment by bnewbold** [comment, marked as answer]
Looking at the parent posts in that thread, it seems like the `facets` list is empty. As described in the linked blog post, specifically in the "Mentions and Links" section, for hyperlinks you need to include facets with the correct type, the URL itself, and the starting and ending byte offsets (counting by UTF-8 bytes).
